packages/react/accordion/src/Accordion.tsx

```diff
@@ -99,9 +101,13 @@ const AccordionSingle = React.forwardRef((props, forwardedRef) => {
     <AccordionValueProvider
       value={value ? [value] : []}
       onItemOpen={setValue}
-      onItemClose={() => setValue('')}
+      onItemClose={() => !preventAllClosed && setValue('')}
```

Comment:
This is the bit where the `preventAllClosed` is implemented for `type="single"`.